hphp/runtime/vm/jit/smashable-instr-arm.h

```diff
@@ -17,6 +17,7 @@
 #ifndef incl_HPHP_JIT_SMASHABLE_INSTR_ARM_H_
 #define incl_HPHP_JIT_SMASHABLE_INSTR_ARM_H_
 
+#include "hphp/runtime/vm/jit/cg-meta.h"
```

Comment:
Why did you add this?  This header doesn't seem to require the definition of CGMeta.